src/chrome-extension/options.ts

```diff
@@ -8,10 +8,22 @@ type RecordingSetup = {
 const recordings = new Map<string, RecordingSetup>();
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
-async function startStreaming(port: number) {
+async function startStreaming(port: number, recordingResizeFactor: number = 1) {
```

Comment:
Is it accepting the same parameters described by type `StartStreamingOptions`?

Reply:
yes

Reply:
then, let's use the `StartStreamingOptions` type